R/animint.R

```diff
@@ -1633,7 +1638,138 @@ animint2dir <- function(plot.list, out.dir = tempfile(),
       stop("missing first selector variable")
     }
   }
-
+  
+  ## Compute domains of different subsets, to be used by update_scales
+  ## in the renderer
+  compute_domains <- function(built_data, axes, geom_name,
+                             var, split_by_panel){
+    # Different geoms will use diff columns to calculate domains for
+    # showSelected subsets. Eg. geom_bar will use 'xmin', 'xmax', 'ymin',
+    # 'ymax' etc. while geom_point will use 'x', 'y'
+    domain_cols <- list(bar=c(paste0(axes, "min"), paste0(axes, "max")),
```

Comment:
this is fine but make sure to stop with an error if there is some geom which is not recognized